presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergMetadata.java

```diff
@@ -289,6 +291,12 @@ public void renameSchema(ConnectorSession session, String source, String target)
         metastore.renameDatabase(new HiveIdentity(session), source, target);
     }
 
+    @Override
+    public void setSchemaAuthorization(ConnectorSession session, String source, PrestoPrincipal principal)
```

Comment:
Can you please also add for iceberg connector, like you did for Hive.

Does iceberg supports sql-standard?

Maybe we should not implement this for iceberg for now?

Reply:
I'm not sure I follow with this one. This is all that is needed for Iceberg, right? I did not actually change the HiveConnector, I added it to the metastore and the metadata (just as I have done here). Apologies if I'm missing something.

Reply:
In hive we have sql-standard access control which controls who can change the schema owner. How is it with Iceberg? As I understand after your change, anyone can change the schema owner. Am I right? If so, then maybe it is better to not to add this security leak to iceberg just yet. WDYT?

Reply:
Oh I see what you mean now.

The code here just defers to whatever the referenced HiveMetastore implementation can do.
The real Hive Metastore is one implementation, there's also the filebased metastore, etc. I added the right code to all metastore implementations (and the default is to deny this operation).

The metastore implementations will deny the change if it is not supported, and allow it if is supported, so it should be OK as is!